src/main.js

```diff
@@ -1642,7 +1642,9 @@ const DiscoveryFeedApplication = new Lang.Class({
         this._installedAppsChangedId = -1;
         this._monitorAddedSignalId = -1;
         this._monitorRemovedSignalId = -1;
-        this._monitorWorkareaChangedSignalId = -1;
```

Comment:
Not sure why this got removed here

Reply:
It also does not seem like d-f is using Wnck already, so I'm not too sure this commit is really needed anymore

Reply:
Probably rebase errors. I'll take another look.